meta-lmp-bsp/wic/sdimage-imx8-sota.wks.in

```diff
@@ -2,19 +2,21 @@
 # long-description:
 # Create an image that can be written onto a SD card using dd for use
 # with i.MX 8 SoC family
-# It uses u-boot + other binaries gathered together on imx-boot file
+# SPL binary is just renamed imx-boot (contains: u-boot-spl-nodtb.bin; dt-spl.dtb; ddr firmware binaries)
+# u-boot partition contains rawcopy of U-Boot FIT image (contains: bl31.bin; tee.bin; u-boot.bin; u-boot.dtb)
 #
 # The disk layout used is:
-#  - ---------- -------------- --------------
-# | | imx-boot |     boot     |    rootfs   |
-#  - ---------- -------------- --------------
-# ^ ^          ^              ^
-# | |          |              |
-# 0 |        4MiB          16MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
+#  - ---------- -------------- --------------------
+# | | SPL  | u-boot  |     boot     |    rootfs   |
+#  - ---------- -------------- --------------------
+# ^ ^      ^         ^              ^
+# | |      |         |              |
+# 0 |      1MiB      8MiB           16MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
```

Comment:
we can use the same layout not only for SD, but for eMMC also. This requires changes of both uuu scripts:
`bootloader.uuu.in` and `full_image.uuu`.
In `bootloader.uuu.in` we can write bootloader, but instead of `flash` cmd use something like:
```
FB: ucmd mmc write ${fileaddr} 800 ${filesz}
```

Reply:
or we can just still use boot0/boot1 hw paritions for eMMC as before
@mike-scott @ricardosalveti @ldts please decided

Reply:
Such layout obviously requires different value for `CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR` in U-Boot